net/freeradius3/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=            freeradius3
+PLUGIN_VERSION=         0.0.1
+PLUGIN_COMMENT=         Freeradius3
+PLUGIN_DEPENDS=         talloc python2 gdbm
```

Comment:
Hm, these were the dependencies when installing freeradius, don't know if it's required here also